docs/maps/geojson-upload.asciidoc

```diff
@@ -0,0 +1,43 @@
+[role="xpack"]
+[[geojson-upload]]
+== Upload GeoJSON data
+*Elastic Maps* makes it easy to import geospatial data into the Elastic Stack.
+Using the *GeoJSON Upload* feature, you can drag and drop your points,
+shape files, and features directly into {es}, and then use them as layers 
```

Comment:
Is "shape files" the right wording here? I interpret "shape files" as the [ESRI vector file format](https://doc.arcgis.com/en/arcgis-online/reference/shapefiles.htm#targetText=A%20shapefile%20is%20an%20Esri,and%20contains%20one%20feature%20class.), not a file of geojson.

Reply:
I agree. I caught another one of these but missed this one. Nice catch, updated!